twisted/twistedchecker
# Allow reserverd __XXX__ variable names

Problem
======

This is a re-submission for #77

The problem is that names like `__dict__ , __version__, __etc__` should always be accepted.

Changes
=======

I have updated the pylint regex expression for attribute names to accept these names.

I place all good and bad examples in a single file so that it is much easier to review/check the values.

How to test
=========

Check the new sample file together with the result and check that it is ok.

try to come with other examples which does not pass the test.

thanks!
